src/components/navigation/left-nav.tsx

```diff
@@ -43,7 +54,19 @@ export class LeftNavComponent extends BaseComponent<IProps, {}> {
           aria-labelledby={this.getTabId(activeSectionIndex)}
           aria-hidden={ui.leftNavExpanded}
         >
-          <LeftNavPanelComponent section={activeSection} isGhostUser={this.props.isGhostUser} />
+          {sections.map((section, index) => {
+            return (
+              this.state.tabLoadAllowed[index]
+              ? <div
+                  id={this.getContainerId(index)}
```

Comment:
This container div is what we look for in the Cypress tests to find the correct open button to load a workspace.  I'm referencing it by an index number in the id.  Maybe a more desirable way to do this?  Use the workspace name?  Put it in a data-test attribute?